pytest.md

```diff
@@ -1,3 +1,18 @@
+## 2021-07-08
+### Unit testing on a function whether an exception is properly raised
+- Using `pytest.raises`
+
+```python
+import pytest
+
+def test_func_division_error():
+    with pytest.raises(ZeroDivisionError):
+        9/0
```

Comment:
if we a mocking a function, and we expect the function to throw error, we can do

with patch("function_to_mock") as f:
  f.side_effect = ZeroDivisionError